Makefile

```diff
@@ -174,9 +174,13 @@ endif
 #   make test-extended SUITE=core
 #   make test-extended SUITE=conformance FOCUS=pods
 SUITE ?= conformance
-FOCUS ?= .
+ifneq ($(strip $(FOCUS)),)
```

Comment:
Their [CLI documentation](https://onsi.github.io/ginkgo/#the-ginkgo-cli) claims it takes the value of this flag as a regex.